shared_types/src/chat_message.rs

```diff
@@ -22,6 +30,7 @@ pub struct ChatMessage {
 #[derive(Debug, Clone, Serialize, Deserialize, PartialEq, Eq)]
 pub struct ChatMessageContainer {
     pub destination: ChatDestination,
+    // TODO: @ion maybe even better to change this to messages: Vec<ChatMessage>
```

Comment:
We should leave this in and do this at one point. Because having it send every message individually is actually not great.